server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -107,6 +112,85 @@ public int getBalancerComputeThreads()
     return balancerComputeThreads;
   }
 
+  @JsonProperty
+  public Set<String> getKillDataSourceWhitelist()
+  {
+    return killDataSourceWhitelist;
+  }
+
+  @Override
+  public String toString()
+  {
+    return "CoordinatorDynamicConfig{" +
+           "millisToWaitBeforeDeleting=" + millisToWaitBeforeDeleting +
+           ", mergeBytesLimit=" + mergeBytesLimit +
+           ", mergeSegmentsLimit=" + mergeSegmentsLimit +
+           ", maxSegmentsToMove=" + maxSegmentsToMove +
+           ", replicantLifetime=" + replicantLifetime +
+           ", replicationThrottleLimit=" + replicationThrottleLimit +
+           ", balancerComputeThreads=" + balancerComputeThreads +
+           ", emitBalancingStats=" + emitBalancingStats +
+           ", killDataSourceWhitelist=" + killDataSourceWhitelist +
+           '}';
+  }
+
+  @Override
+  public boolean equals(Object o)
```

Comment:
If someone adds a new config item later it would be very easy to forget to update the hashCode and equals